CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning][].
 -   Multipolygons are now handled correctly (#93)
 -   Legend order is now deterministic (#143)
 -   Images no longer normalised by default (#150)
+-   Shapes can be colored by categorical variable with the `group` and `palette` arguments (#153)
```

Comment:
Please make clear that `groups` is used to _filter_ the data and `palette` is used to color the (potentially remaining) categories. Also, what happens if I add a palette argument to a continous variable?